src/electionguard/serializable.py

```diff
@@ -22,7 +24,7 @@
 READ: str = "r"
 JSON_PARSE_ERROR = '{"error": "Object could not be parsed due to json issue"}'
 # TODO Issue #??: Jsons library incorrectly dumps class method
-FROM_JSON_FILE = '"from_json_file": {}, '
+KEYS_TO_REMOVE = ["from_json", "from_json_file", "from_json_object"]
```

Comment:
This is an extremely frustrating part of `jsons`. It get confused on class_methods and these keys need to get removed or they show up in the exported object.